### PR TITLE
WK-201 style form input controls

### PR DESCRIPTION
**Pull request by aditudorache**
This PR styles the input elements in the Location edit form. The unittests will be implemented in a later story.

### CODE REVIEW

File: src/views/manage/ManageFormStyle.jsx [hunk 1]
```diff
@@ -8,12 +14,33 @@ export const ManageFormStyle = styled.form`
...
+export const FixedRow = styled(Row)`
+  width: 100%;
+  position: absolute;
+  bottom: 0;
+  border-top: 4px solid ${themeColor('tint', 'level3')};
+`;
+
+export const ButtonsColumn = styled(Column)`
+  background-color: ${themeColor('tint', 'level1')};
+  padding: 8px 0 12px;
```
Comment (janjaap) [inline src/views/manage/ManageFormStyle.jsx]: `themeSpacing()`?

File: src/views/manage/ManageFormStyle.jsx [hunk 2]
```diff
@@ -8,12 +14,33 @@ export const ManageFormStyle = styled.form`
...
+  align-items: flex-start;
```
Comment (janjaap) [inline src/views/manage/ManageFormStyle.jsx]: Isn't this default?

File: src/views/manage/components/DatePickerInput.jsx [hunk 3]
```diff
@@ -2,10 +2,41 @@ import React, { useState, useEffect, useRef } from 'react';
...
+import { Calendar } from '@datapunt/asc-assets';
+import { Icon, styles, themeColor } from '@datapunt/asc-ui';
+
+const DatePickerStyle = styled.div`
+  position: relative;
+  & > .react-datepicker-wrapper > .react-datepicker__input-container > input {
+    border: solid 1px ${themeColor('tint', 'level4')};
+    border-radius: 0;
+    box-sizing: border-box;
+    padding: 10px;
```
Comment (janjaap) [inline src/views/manage/components/DatePickerInput.jsx]: `themeSpacing()`?

File: src/views/manage/components/DatePickerInput.jsx [hunk 4]
```diff
@@ -2,10 +2,41 @@ import React, { useState, useEffect, useRef } from 'react';
...
+    width: 100%;
+  }
+
+  & > ${styles.IconStyle} {
+    position: absolute;
+    top: 10px;
```
Comment (janjaap) [inline src/views/manage/components/DatePickerInput.jsx]: `themeSpacing()`?

File: src/views/manage/ManageFormStyle.jsx (hunk 1 above)
Comment (aditudorache) [reply src/views/manage/ManageFormStyle.jsx]: 👍🏻

File: src/views/manage/ManageFormStyle.jsx (hunk 2 above)
Comment (aditudorache) [reply src/views/manage/ManageFormStyle.jsx]: good point! thanks!

File: src/views/manage/components/DatePickerInput.jsx (hunk 3 above)
Comment (aditudorache) [reply src/views/manage/components/DatePickerInput.jsx]: not possible here, the InputStyle in de `ASC` has a padding of 10, trying to keep that here.

File: src/views/manage/components/DatePickerInput.jsx (hunk 4 above)
Comment (aditudorache) [reply src/views/manage/components/DatePickerInput.jsx]: not appropriate here

**Pull request merged**